lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -251,6 +258,20 @@ system_default_switchport_shutdown:
   config_get_token: ['/^system default switchport shutdown$/']
   # default_value: n/a. This is a user-configurable system default.
 
+vlan_mapping:
+  _exclude: [/N(3|5|6|9)K/]
```

Comment:
We could probably do something like this for all of the platforms.. just thinking outloud...

`[/N(3|5|6|9|K?/]`